### ISSUE TITLE
yaml front matter

### ISSUE THREAD

**Issue by dindinet** [classeur/classeur#45]
Hi I am having trouble getting the The toYaml helper 
to work. When I look at the post on github I see
Error: SyntaxError: Unexpected token { 

I paste the
`---`
{{#toYaml file.content.properties}}{{/toYaml}}
`---`
 into the Settings>HANDLEBARS HELPERS

Is this the correct place to paste the handlebar snippet?

Regards

**Comment by benweet** [comment]
No, you have to create or modify a template in the the "Manage template" dialog (in the export/publish dialog, not in the settings).

I'll keep the ticket open until we have a clearer explanation in the app and a more intuitive UI.

**Comment by dindinet** [comment]
hi benweet

I have an Export dialog not an Export/Publish Dialog

I added a Plain Text template for Text. If I export my document it works I get a text file with yaml frontmatter added.

When I publish to Github I only get the body text of the post.

Regards

**Comment by benweet** [comment]
Did you use the same template during export and publish?
Can you provide the template and the url of the file on GitHub?

**Comment by dindinet** [comment]
Hi benweet

This is the menu I see
![](http://goo.gl/bf3APx)
this is what I see after pressing Export
![](http://goo.gl/wClLG3)

This is the template.
`---`
{{#toYaml file.content.properties}}{{/toYaml}}
`---`
{{{file.content.text}}}

These are the settings for the doc on classeur
![](http://goo.gl/FwQISA)
This is the file on GitHub https://github.com/dindigit/dindigit.github.io/blob/master/_posts/2015-11-20-Fish-Adding_Post.md

**Comment by benweet** [comment]
You may have created the blog post before modifying the template, so it keeps using the old template. Can you try to delete the entry in the Post to Blog dialog, and recreate the post?

**Comment by dindinet** [comment]
Boom! It worked.

Great. New post published with file props as YAML.
https://github.com/dindigit/dindigit.github.io/blob/master/_posts/2015-11-20-Fox-Post.md
